package.json

```diff
@@ -29,50 +29,39 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch:lib": "watch 'npm run lint && npm test' ./lib"
   },
   "dependencies": {
-    "async": "1.5.2",
-    "body-parser": "~1.11.0",
+    "async": "2.6.1",
+    "body-parser": "~1.18.3",
     "command-shell-lib": "1.0.0",
-    "express": "~4.11.2",
-    "jison": "0.4.17",
-    "logops": "1.0.0",
-    "moment": "~2.19.4",
+    "express": "~4.16.4",
+    "jison": "0.4.18",
+    "logops": "2.1.0",
```

Comment:
I fully agree with you. Using logops 1.0.0 is not so critical since it does not introduce any vulnerability. I will update this PR and we can create a new issue as you suggest.

Reply:
Fixed with ac2e855ba3d58c3d0bea4b872285e9f4cf3bbedc. Now unit tests are passed for all versions of Node.js (4,6, 8 and 10)

Reply:
New issue opened in https://github.com/telefonicaid/iotagent-node-lib/issues/689

Reply:
As you can see, the fact of not using the latest version of logops does not introduce any vulnerabilities:

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN iotagent-node-lib@2.7.0-next No license field.
```
added 280 packages from 768 contributors and audited 690 packages in 60.07s
found 0 vulnerabilities
```